readthedocs/subscriptions/event_handlers.py

```diff
@@ -114,7 +114,7 @@ def subscription_canceled(event):
         log.info("Stripe subscription not found.")
         return
 
-    organization = stripe_subscription.customer.rtd_organization
+    organization = getattr(stripe_subscription.customer, "rtd_organization", None)
```

Comment:
```suggestion
    # Using `getattr` to avoid the `RelatedObjectDoesNotExist` exception
    # when the subscription doesn't have an organization attached to it.
    organization = getattr(stripe_subscription.customer, "rtd_organization", None)
```